packages/ui/src/components/cell/cell.native.tsx

```diff
@@ -87,11 +98,10 @@ export function Cell({ title, subtitle, onPress, Icon, ...props }: CellProps) {
           )}
         </Box>
       </Box>
-      {props.variant !== 'radio' && props.variant !== 'switch' && (
+      {_variant === 'active' && (
         <ChevronRightIcon color={theme.colors['ink.text-primary']} variant="small" />
       )}
-      {/* TODO: Temp placeholder - implement custom radio button */}
-      {props.variant === 'radio' && <></>}
+      {props.variant === 'radio' && <RadioButton isSelected={props.isSelected} />}
```

Comment:
Adding to the `Cell` variants here, but wondering if we should just have a better composed `Cell` to work with? The `active` version is still very much a duplicate of `ItemLayout`, but `ItemLayout` is more flexible. I added a conditional here to cover when there is no `Icon`.

Reply:
Alternatively, maybe we just want to kill `ItemLayout` and `Flag` for mobile, maybe `Cell` is all we need? I know I keep asking this... 🤣 don't laugh @pete-watters.

Reply:
Find it hard to believe that we won't use Flag again, it's such a common layout pattern? Cell can use Flag, no?

Reply:
Yeah, hoping we can refactor Cell to use it, but I don't want to step on Pete's toes bc he was working on it. @pete-watters you want me to do?

Reply:
I can add `Flag` into `Cell`, no problem. I did it already so I can do it again. See this discussion @kyranjamie : https://github.com/leather-io/mono/pull/416#discussion_r1740683610 

We can use `Flag` in `Cell` but that's the only place to use it. We can also manipulate `ItemLayout` to fit into the usage of `Cell`. But then you may ask, what's the point as we use `Cell` everywhere and the only place we use `Flag` and `ItemLayout` will be in `Cell`. 

Of course they could be of use in the future but at this point in time we are just moving code around and not adding specific value. I think we can work smarter and build features during which we figure out what components we need rather than building loads of components that we might need or not